Gemfile

```diff
@@ -32,7 +33,7 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem 'quiet_assets', '~> 1.1.0'
-  gem 'govuk-lint', '0.5.0'
+  gem 'govuk-lint'
```

Comment:
I was thinking that without the pinned version, we could upgrade this at any time with `bundle update govuk-lint` (and double check the current version in Gemfile.lock). What do you think?